stdlib/public/Platform/CMakeLists.txt

```diff
@@ -20,6 +21,11 @@ else()
   set(swift_platform_sources
       Glibc.swift
       ${swift_platform_sources})
+  if(SWIFT_HOST_VARIANT_SDK STREQUAL "FREEBSD")
```

Comment:
Could you add the file unconditionally?  This way cross-compilation from Linux to FreeBSD will just work.